src/components/molecules/NavBarScheduleWeather/WeatherBackground/BlackRockWeatherBackground.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { PropsWithChildren, useState } from "react";
+import useInterval from "react-use/lib/useInterval";
+import classNames from "classnames";
+import dayjs from "dayjs";
+import timezone from "dayjs/plugin/timezone";
+import utc from "dayjs/plugin/utc";
+
+import { ContainerClassName } from "types/utility";
+
+import "./WeatherBackground.scss";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
```

Comment:
The plugin for timezone manipulations `date-fns-tz` looks complicated. I've spent a significant time with it and didn't achieve what I wanted.
For now, I refactored the common code from `PlayaTime.tsx` that also uses dayjs `timezone` plugin to use in `BlackRockWeatherBackground.tsx`. So there was one usage of dayjs `timezone` plugin and it remained that way:)
https://github.com/sparkletown/sparkle/pull/2092/commits/56df586391d1c7c342380c6cf2af2606ae8f3579

Reply:
> So there was one usage of dayjs timezone plugin and it remained that way:)

Good enough, we'll know where to get it.

[OOS] On this:

> The plugin for timezone manipulations date-fns-tz looks complicated. I've spent a significant time with it and didn't achieve what I wanted.

Did you find any help? Since we might be migrating from `dayjs` it might be prudent to find a solution for `date-fns`  abstract it in `src/util` function for everyone else to reuse or in exteme case re-examine if time zones are blocker for it.

Reply:
Surely it can be made to work somehow - the lib is quite popular and robust. I just didn't manage to make it to...